quickstart/configure-mayastor.md

```diff
@@ -1,45 +1,45 @@
+| Asynchronous Disk I/O \(AIO\) | Device File | aio:///dev/sdx |
 | io\_uring | Device File | uring:///dev/sdx |
-| RAM drive | Custom | malloc:///malloc0?size\_mb=1024 |
+| Disk(non PCI) with disk-by-guid reference <i>(Best Practice)</i> | Device File | aio:////dev/disk/by-uuid/<uuid_number> OR uring:////dev/disk/by-uuid/<uuid_number> |
 
-Once a Mayastor node has created a pool it is assumed that it henceforth has exclusive use of the associated block device; it should not be partitioned, formatted, or shared with another application or process. Any pre-existing data on the device will be destroyed.
+Once a node has created a pool it is assumed that it henceforth has exclusive use of the associated block device; it should not be partitioned, formatted, or shared with another application or process. Any pre-existing data on the device will be destroyed.
 
 {% hint style="warning" %}
 A RAM drive isn't suitable for use in production as it uses volatile memory for backing the data. The memory for this disk emulation is allocated from the hugepages pool. Make sure to allocate sufficient additional hugepages resource on any storage nodes which will provide this type of storage.
 {% endhint %}
 
 ### Configure Pools for Use with this Quickstart
 
-To continue with this quick start exercise, a minimum of one pool is necessary, created and hosted on one of the Mayastor nodes in the cluster. However, the number of pools available limits the extent to which the synchronous n-way mirroring feature \("replication"\) of Persistent Volumes can be configured for testing and evaluation; the number of pools configured should be no fewer than the desired maximum replication factor of the PVs to be created. Also, while placing data replicas ensure that appropriate redundancy is provided. Mayastor's control plane will avoid locating more than one replica of a volume on the same Mayastor node. Therefore, for example, the minimum viable configuration for a Mayastor deployment which is intended to test 3-way mirrored PVs must have three Mayastor Nodes, each having one Mayastor Pool, with each of those pools having one unique block device allocated to it.
+To get started, it is necessary to create and host at least one pool on one of the nodes in the cluster. The number of pools available limits the extent to which the synchronous n-way mirroring feature (replication) of PVs can be configured; the number of pools configured should be equal to or greater than the desired maximum replication factor of the PVs to be created. Also, while placing data replicas ensure that appropriate redundancy is provided. Mayastor's control plane will avoid placing more than one replica of a volume on the same node. For example, the minimum viable configuration for a Mayastor deployment, which is intended to implement 3-way mirrored PVs must have three nodes, each having one DiskPool, with each of those pools having one unique block device allocated to it.
```

Comment:
There should be no comma after _Mayastor deployment_ and _before which is intended_

It changes the meaning of the sentence.  The Mayastor deployment is the subject of the specific intention.  i.e. the particular mayastor deployment from the example is intended to support 3 way mirrors, so needs three pools.  Another might be intended to support four way, and so would need four pools.  As written here, the minimum viable configuration is always to have three nodes, which is intended to support three way mirrors.

Reply:
That makes sense. Will update it.

Reply:
"synchronous N-way mirroring (replication) of PVs"
 Remove "feature" and change n-way to N-way.